model/src/test/kotlin/licenses/ResolvedLicenseInfoTest.kt

```diff
@@ -143,6 +149,177 @@ class ResolvedLicenseInfoTest : WordSpec() {
                 filteredResolvedLicenseInfo.licenses should containLicensesExactly(mit, bsd)
             }
         }
+
+        "filterLicenseResolutions()" should {
+            "remove license that is resolved by a license rule violation resolution" {
+                val resolvedLicenseInfo = ResolvedLicenseInfo(
+                    id = Identifier.EMPTY,
+                    licenseInfo = mockk(),
+                    licenses = listOf(
+                        ResolvedLicense(
```

Comment:
To remove clutter and allow focusing on essentials, I would extract the creation of a `ResolvedLicense` to a helper function that expects the license as parameter. This should shorten the test cases significantly.